config.yaml

```diff
@@ -8,3 +8,8 @@ options:
     description: |
       Request roles instead of extensions
     type: boolean
+  sleep_interval:
+    default: 0
+    description: |
+      How long to sleep between writes
```

Comment:
Please indicate the unit: seconds, minutes, milliseconds? Tnx!

Reply:
Added unit